addons/common/functions/fnc_canFire.sqf

```diff
@@ -38,6 +38,7 @@ alive _unit
         _weapon != ""
         && {!("fake" in toLower _weapon)}
         && {_ignoreAmmo || {_ammoCount > 0} || {_weapon isKindOf ["CarHorn", configFile >> "CfgWeapons"]}}
+        && {!(_weapon isKindOf ["Laserdesignator_mounted", configFile >> "CfgWeapons"] && {CBA_missionTime < _unit getVariable [QGVAR(nextFireTime), 0]})}
```

Comment:
```suggestion
        && {CBA_missionTime >= _unit getVariable [QGVAR(nextFireTime), 0]}
```
Would work the same and support future setting of a manual lockout for any weapon/unit. Also, move below the reload check.